statsmodels/graphics/tests/test_gofplots.py

```diff
@@ -686,3 +686,16 @@ def test_axis_order(close_figures):
     y_range = np.diff(ax.get_ylim())[0]
     x_range = np.diff(ax.get_xlim())[0]
     assert x_range < y_range
+
+@pytest.mark.matplotlib
+def test_qqplot_2samples_labels():
+    try:
```

Comment:
You shouldn't need this.  You can safely import here since we have the decorator that skips if MPL isn't installed.